Kernel/FileSystem/SysFS/DirectoryInode.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/StringView.h>
+#include <Kernel/FileSystem/SysFS/DirectoryInode.h>
+#include <Kernel/Sections.h>
+#include <Kernel/Time/TimeManagement.h>
+
+namespace Kernel {
+
+ErrorOr<NonnullLockRefPtr<SysFSDirectoryInode>> SysFSDirectoryInode::try_create(SysFS const& sysfs, SysFSComponent const& component)
+{
+    return adopt_nonnull_lock_ref_or_enomem(new (nothrow) SysFSDirectoryInode(sysfs, component));
+}
+
+SysFSDirectoryInode::SysFSDirectoryInode(SysFS const& fs, SysFSComponent const& component)
+    : SysFSInode(fs, component)
+{
+}
+
+SysFSDirectoryInode::~SysFSDirectoryInode() = default;
+
+InodeMetadata SysFSDirectoryInode::metadata() const
+{
+    // NOTE: No locking required as m_associated_component or its component index will never change during our lifetime.
+    InodeMetadata metadata;
+    metadata.inode = { fsid(), m_associated_component->component_index() };
+    metadata.mode = S_IFDIR | S_IRUSR | S_IRGRP | S_IROTH | S_IXOTH;
+    metadata.uid = 0;
+    metadata.gid = 0;
+    metadata.size = 0;
+    metadata.mtime = TimeManagement::boot_time();
+    return metadata;
+}
```

Comment:
Shouldn't clang format have inserted a new line after this `}`?

Reply:
Fixed. I don't know why clang-format didn't fix it though.